### PR TITLE
Add dataparser for handelsbanken / BEC banks

### PR DESCRIPTION
**Pull request by larsbaunwall**
Add support for Handelsbanken / BEC export in the format [Date;Text;Amount;Saldo;Dummy]

### CODE REVIEW

**Comment by Laumania** [comment]
Nice and thanks! Always good with PR's - I'll look at it later and merge it.

**Comment by Laumania** [comment]
@larsbaunwall Sorry, haven't merged yet, as I haven't had the time - busy at work - I will look at it soon how ever :)

Again, thanks for you pull request.

**Comment by Laumania** [comment]
Hi @larsbaunwall 
Again, sorry for the delay. I looked at your changes and let me say first, nicely done - you keep indenting stuff as the rest of the project - the OCD approves :P

However, could I ask you to add a Unittest for your new DataParser too?

File: Dinero.Tools.Difference/Dinero.Tools.Difference.Core/DataParsers/HandelsbankenDataParser.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                    Saldo   = shbEntry.Saldo
+                };
+                result.Add(entryModel);
+            }
+
+            return result;
+        }
+
+        [DelimitedRecord(";"), IgnoreEmptyLines(), IgnoreFirst()]
+        private class SparNordEntry
```
Comment (Laumania) [inline Dinero.Tools.Difference/Dinero.Tools.Difference.Core/DataParsers/HandelsbankenDataParser.cs]: @larsbaunwall Guess you missed renaming 'SparNordEntry'? I know it's private and all, but the name is miss leading.

**Review by Laumania — COMMENTED** [review]
Never tried this "Review" thing before to be honest. Seems pretty smart :)

**Comment by Laumania** [comment]
@larsbaunwall Hi Lars, did you have time to look at my comments?
I really want to have a Unittest added with a sample of the BEC file format, to make sure it doesn't break the logic behind finding the right parser for the uploaded file.

**Comment by Laumania** [comment]
@larsbaunwall I'm not sure if you don't see these messages or you just don't have time to look into this. Can you please reply, if you get these and just haven't had time to look into it?

I would very much like to have this PR merged in :)

**Comment by larsbaunwall** [comment]
Hi Mads.
You just saw your messages - have been occupied elsewhere for the last month or so.
I will look into your comments as soon as possible 👍

**Comment by larsbaunwall** [comment]
Just committed the changes - hope you like them :)

**Comment by Laumania** [comment]
@larsbaunwall No problem, no rush - just wanted to make sure you got the notification, so at least that wasn't the reason why :)

Very nice, I'll take a look at your changes soon - thanks for your pull request once again!

**Comment by Laumania** [comment]
@larsbaunwall Looked good! I see 2 unittests fails, but they do that in master too, so it's not your fault. I'll look at it in the master branch.

Thank you so much for your Pull Request - keep 'em coming :)

**Pull request merged**